packages/api-core/src/me-sites-plugins/fetchers.ts

```diff
@@ -0,0 +1,6 @@
+import { wpcom } from '../wpcom-fetcher';
+import type { PluginsResponse } from './types';
+
+export async function fetchUserSitesPlugins(): Promise< PluginsResponse > {
+	return await wpcom.req.get( { path: '/me/sites/plugins', apiVersion: '1.1' } );
```

Comment:
if we only need `sites`, it would be better to return it directly instead of the whole response 🙂
```js
const { sites } = await wpcom.req.get( { path: '/me/sites/plugins', apiVersion: '1.1' } );
return sites;
```

Reply:
I don't think this is a substantial improvement.